.gitignore

```diff
@@ -0,0 +1,11 @@
+node_modules
+coverage
+.DS_Store
+.nyc_output
+.idea
+.env
+.vscode
+
+github_token
+helm/**/*.tgz
+jsHelper.js
```

Comment:
Is this a common file to ignore in projects?

Reply:
No. I don't know why this is ignored when it's not referenced anywhere

Reply:
> Is this a common file to ignore in projects?

good question, it's not by convention and something I started adding for convenience which is still yet to be discussed.
Happy to remove this if this is not something that is useful to everyone. Thoughts team?

Reply:
Not sure it's a good idea to have files here and in the linter config, that don't exist and may be created or have code added in the future?

Reply:
Absoltely these lines should be removed